src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -879,4 +882,140 @@ protected static async Task<ImmutableArray<string>> GetAllMatchingGlobalAliasNam
 
         return result.ToImmutableAndClear();
     }
+
+    protected static async ValueTask DiscoverImpliedSymbolsAsync(
+        TSymbol symbol, Solution solution, ArrayBuilder<ISymbol> symbolBuilder, CancellationToken cancellationToken)
+    {
+        var name = symbol.Name;
+
+        var symbolSet = PooledHashSet<ISymbol>.GetInstance();
+        var documentQueue = new ConcurrentQueue<Document>();
+        Task? documentProcessingTask = null;
+
+        foreach (var project in solution.Projects)
+        {
+            var compilation = await project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
uh.  this is forcing compilations on *every* project in the compilation. That's going to be a massive perf cost.  why is this being done?

Waiting to hear back on this before proceeding.

Reply:
That was a leftover, which was not used anywhere in the code below. Removing.